spk/homeassistant/src/installer.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python3"
+PATH="${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+GROUP="users"
```

Comment:
If you need access to the filesystem (outside of the users home dir), then use the users group. If access is not needed, use nobody. Because I didn't know about that component, I suggested to use the nobody group.

But, if you want to make that component useable, maybe best to add a way to set that downloadpath, because otherwise, people will have to dig into the config to change the path manually. That's why I asked if the path could be set via the web interface.
If you can't set it via the interface, maybe add a wizard (there's a lot of packages using the wizard to do that, take a look at couchpotato for instance)